config/config_handler.go

```diff
@@ -198,9 +230,10 @@ func (ch *pdConfHandler) run() {
 
 			ch.reloadFunc(ch.curConf.Load().(*Config), newConf)
 			ch.curConf.Store(newConf)
-			logutil.Logger(context.Background()).Info("PDConfHandler update config successfully",
-				zap.String("fromVersion", ch.version.String()), zap.String("toVersion", version.String()))
 			ch.version = version
+			logutil.Logger(context.Background()).Info("PDConfHandler update config successfully",
+				zap.String("fromVersion", ch.version.String()), zap.String("toVersion", version.String()),
```

Comment:
Now ch.version and version is same.... So the log seems be wrong.

Reply:
My fault, it is updated now.